.github/workflows/build_test.yml

```diff
@@ -28,18 +28,18 @@ jobs:
     - name: Test
       run: dotnet test --configuration Release --no-restore --verbosity normal
     - name: Pack
-      if: ${{ github.event_name == 'push' }}
```

Comment:
why you change this build script ?

Reply:
It's solely for testing purpose, will revert the change once i'm done with testing.